projects/plugins/jetpack/tests/php/modules/geo-location/test_class.jetpack-geo-location.php

```diff
@@ -375,7 +375,7 @@ private function get_instance_with_mock_private_post() {
 	/**
 	 * @param string[] $additional_mock_methods
 	 * @param boolean  $disable_constructor
-	 * @return Jetpack_Geo_Location|PHPUnit_Framework_MockObject_MockObject
+	 * @return Jetpack_Geo_Location&\PHPUnit\Framework\MockObject\MockObject
```

Comment:
What does the `&` do? I see a brief reference here, but it's not clear to me:
https://github.com/phan/phan/wiki/About-Union-Types

Reply:
https://www.php.net/manual/en/language.types.type-system.php#language.types.type-system.composite.intersection may also be helpful.

"Foo&Bar" indicates that the object is both a Foo and a Bar at the same time. This can happen, for example, if one or both of Foo and Bar are names of interfaces.